website/content/docs/configuration/storage/postgresql.mdx

```diff
@@ -79,6 +79,12 @@ to disable SSL.
   PostgreSQL 9.5 or later. Set to `true` if the database user does not have
   the required permissions; otherwise, OpenBao will fail to start.
 
+- `max_connect_retries` `(int: 1)` - Maximum number of retries to perform
+  when waiting for the database to be active. This uses exponential backoff,
+  starting with 15ms and increasing to 5s between retries. To ensure the
+  connection is active, issues a `SELECT RANDOM() as random` query. Can be
```

Comment:
This is incorrect, we call `db.Ping(...)`, whose behavior isn't specified: https://pkg.go.dev/database/sql#DB.PingContext

> Ping verifies a connection to the database is still alive, establishing a connection if necessary.